examples/finnegans_wake_demo/finnegans-wake-demo-federated.py

```diff
@@ -140,7 +140,7 @@
     # single passage from James Joyce's Finnegan's Wake.
     # The matter of whether encryption makes the passage more or less readable
     # is left to the reader to determine.
-    single_passage_ciphertext, _signature = enrico.encrypt_message(plaintext)
+    single_passage_ciphertext = enrico.encrypt_message(plaintext)
```

Comment:
did we want to rename this to `single_passage_message_kit`. Analogous to the change done in the getting started docs.